### PR TITLE
Upgrade to Polymer 2.x

### PR DESCRIPTION
**Pull request by alshakero**

### CODE REVIEW

File: test/d-b-n_smoke/arrays.html [hunk 1]
```diff
@@ -22,79 +22,99 @@
...
-                            <li>{{item}}</li>
-                        </template>
-                    </ul>
-                    <ul class="people_nested">
-                        <template is="dom-repeat" items="{{model.people_nested}}">
-                            <li>
-                                <template is="dom-repeat" items="{{item}}">
+                        </ul>
+                        <ul class="primitive_people">
+                            <template is="dom-repeat" items="{{model.primitive_people}}">
```
Comment (tomalec) [inline test/d-b-n_smoke/arrays.html]: Wouldn't Polymer replace it with `<ul><dom-repeat><template...`? If so, I would replace it here as well, to have less magic under cover. If not, I would keep it as I like `<tempalte is="` use cases :)

File: test/d-b-n_smoke/simple.html [hunk 2]
```diff
@@ -58,12 +61,13 @@
                 });
                 it('when element is removed, should notify any listeners', function (done) {
                     // change model
+                    debugger
```
Comment (tomalec) [inline test/d-b-n_smoke/simple.html]: Please remove it :)

File: test/d-b-n_smoke/simple.html [hunk 3]
```diff
@@ -58,12 +61,13 @@
...
                     palindromChange(palindrom, [operationObject('remove', '/name')]);
                     setTimeout(function () {
                         // dom-if is an example of a listener
                         expect(document.querySelector("#conditional")).to.not.be.visible;
                         done();
-                    })
+                    }, 100)
```
Comment (tomalec) [inline test/d-b-n_smoke/simple.html]: Why do we need to wait longer?

File: test/d-b-n_smoke/arrays.html (hunk 1 above)
Comment (alshakero) [reply test/d-b-n_smoke/arrays.html]: Wow, I wonder how this passed.  This is probably a mistake.

**Comment by alshakero** [comment]
Merging as fixed all review issues.

**Pull request merged**